bedrock/pocket/templates/pocket/home.html

```diff
@@ -0,0 +1,119 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "pocket/base.html" %}
+
+{% from "macros-protocol.html" import split, picto with context %}
+
+{% block page_title %}Homepage{% endblock %}
+
+{% block page_desc %}{{ ftl('pocket-about-us-page-desc') }}{% endblock %}
+
+{% block pocket_css %}
+  {{ css_bundle('protocol-split') }}
+  {{ css_bundle('protocol-picto') }}
+  {{ css_bundle('pocket-home') }}
+{% endblock  %}
+
+{% block content %}
+{% include 'pocket/includes/platform-nav.html' %}
```

Comment:
**issue (non-blocking, existing bug):** the text is Log In when the link is signup: https://getpocket.com/signup/?ep=1

**suggestion:** link should be https://getpocket.com/login/?ep=1, can be fixed here or followed up on in separate issue

Reply:
🦅👁!